pyworkflow/__init__.py

```diff
@@ -58,7 +58,7 @@
 
 HOME = os.path.abspath(os.path.dirname(__file__))
 
-PYTHON = os.environ.get("SCIPION_PYTHON", 'python')
+PYTHON = os.environ.get("SCIPION_PYTHON", 'python3')
```

Comment:
Please do not hardcode the python version! It should be the environment outside to ensure that 'python' is 'python3'

Reply:
I did this bacause the snap do not package the python there is only python3...but I stopped the snap thing when I was getting a core dump....so it might not justify the change.